src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_decorator.py

```diff
@@ -4,7 +4,6 @@
 # --------------------------------------------------------------------------------------------
 
 import importlib
-from tkinter import FALSE
```

Comment:
The import error is only found on

- Test Rpm Package
- Test Deb Packages Focal
- Test Deb Packages Buster

but not on

- Automation Test (Profile Latest) Python310

This is because we don't include `tkinter` in our built Python in order to save package size:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1464021&view=logs&j=2cfc7b5c-b945-5758-720c-434c292791a8&t=e70effe3-d4e7-5613-ef4d-26b0b8e6ed67&l=2392

```
The necessary bits to build these optional modules were not found:
_bz2                  _curses               _curses_panel      
_dbm                  _gdbm                 _lzma              
_sqlite3              _tkinter              readline           
To find the necessary bits, look in setup.py in detect_modules() for the module's name.
```

CentOS 7's python also has no `tkinter`.

On the other hand, ADO's Python has `tkinter`.